calculator/lexer.py

```diff
@@ -11,8 +72,85 @@ def lex(s):
       * "*" - Multiplication
       * "+" - Addition
       * "-" - Subtraction
+      * "/" - Division
     * The following grouping symbols are supported:
       * "("
       * ")"
     """
-    return []
+
+    # Defining char_before and char_after
+    char_before = None
+    char_after = None
+    # Partial_num is the var that stores my nums until I have to append it to my tokens list
+    partial_num = None
+    # List that stores the tokens
+    tokens = []
+    # Stripping the input line of spaces
+    stripped_line = line
+
+    # Iterating through each index in my stripped_line
+    for i in range(0, len(stripped_line)):
+        # Defining "c" as the char at the index "i" in stripped_line
+        c = stripped_line[i]
+        # Defining char_before but if i is the first index then the index will be invalid
+        if i != 0:
+            # Char_before is the current char's index (i), but subtracting 1 because it is 1 before
```

Comment:
I think this is clear without the comment, please remove.

Reply:
Ok 👍